cpp/subprojects/README.md

```diff
@@ -0,0 +1,64 @@
+# Meson Subprojects
+
+The contents of this directory are exclusively used by the Meson build configuration to
+manage project dependencies. The Meson documentation for this functionality can be found
+at https://mesonbuild.com/Subprojects.html
+
+To summarize, Arrow relies upon other projects to successfully compile and link. In
+the case that those dependencies cannot be found on the host system, Meson by convention
+forces users to place those dependencies in the ``subprojects`` directory at the root
+of the project.
+
+The easiest way to populate subprojects is to use Meson's
+[WrapDB system](https://mesonbuild.com/Wrapdb-projects.html). To illustrate how this
+works, let's take a look at the ``googletest`` library that Arrow depends upon for
```

Comment:
If this is a markdown file, you need to use markdown markup not reST :) Therefore only single backticks
```suggestion
works, let's take a look at the `googletest` library that Arrow depends upon for
```